packages/web/src/components/tracks-table/TracksTable.module.css

```diff
@@ -51,31 +53,61 @@
 }
 
 .tablePlayButton {
+  position: absolute;
```

Comment:
For my edification wondering why we do absolute positioning here

Reply:
just so both this and the hidden icon are in the same spot and can be faded in and out